### PR TITLE
Add dashboard code

### PR DESCRIPTION
**Pull request by maiix**

### CODE REVIEW

File: Mayra/index.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Overpass:300,400,700" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
+</head>
+
+<body>
+  <header>
+    <div class="logo">
+      <img src="images/logo.png" alt="" width="140px">
+    </div>
+      <a href="#"><span class="btn new"><i class="fas fa-plus"></i> New Request</span></a>
```
Comment (perryhoffman) [inline Mayra/index.html]: Using a span as a button totally works, usually though the `<button>` tag is preferred!

File: Mayra/index.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    <div class="header-content">
+      <span><a href="#"><i class="far fa-comments"></i></a></span>
```
Comment (perryhoffman) [inline Mayra/index.html]: These icon links are looking good. The `<span>` tags don't seem to be doing a whole lot, I would recommend removing extra tags like this. You will thank yourself when the app gets much larger!

File: Mayra/index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      <span><a href="#"><i class="far fa-question-circle"></i></a></span>
+      <span><a href="#"><i class="far fa-bell"></i></a></span>
+      <span><a href="#">Mayra Pulido</a></span>
+    </div>
+
+  </header>
+  <section class="content">
+    <div class="col-left">
+      <nav class="left-nav">
```
Comment (perryhoffman) [inline Mayra/index.html:29]: 👍

File: Mayra/index.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                      <span>0</span>
+                      <p>Pending Order Requests</p>
+                    </a>
+                </div>
+                <div class="badge">
+                    <a href="#">
+                      <span>2</span>
+                      <p>Pending Travel Requests</p>
+                    </a>
+                </div>
```
Comment (perryhoffman) [inline Mayra/index.html:86]: Each of these `badge` components looks really good. Semantic HTML and indentation, nice!

File: Mayra/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+}
+.btn {
+  color: #FFF;
+  padding: 6px 12px;
+  border-radius: 3px;
+  font-size: 12px;
+  text-transform: uppercase;
+  box-shadow: 0px 2px 3px rgba(0,0,0,.3);
+}
+.new {
```
Comment (perryhoffman) [inline Mayra/css/styles.css]: Be **very** careful with short generic class names like this. For a small page it is totally fine, but when the app/site gets larger these types of classes aren't ideal. They aren't descriptive enough and are at a high risk of having a naming clash which will cause toooons of confusion.

File: Mayra/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+  background-color: #FAAB18;
+  float: left;
+  margin-left: 30px;
+  font-weight: 300;
+}
+.header-content {
```
Comment (perryhoffman) [inline Mayra/css/styles.css:32]: Class naming like this is much better!

File: Mayra/css/styles.css [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+  color: #414141;
+  text-transform: uppercase;
+  border-bottom: 1px solid #EEEEEE;
+}
+.box-content{
+  max-height: 300px;
+  overflow: auto;
+  background: #FFF;
+}
+.row {
```
Comment (perryhoffman) [inline Mayra/css/styles.css]: I told kitty the same thing here,  it isn't ideal to overwrite styles of bootstrap/any framework as it makes that change  EVERYWHERE in the app. A better solution could be to make a modifier class like `.row--box-size` and just add it where you need it!

File: Mayra/css/styles.css [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+  color: #FFF;
+  text-transform: uppercase;
+  font-size: 11px;
+}
+.left-nav a:hover {
+  background-color: rgba(0, 0, 0, 0.1);
+  border-right: 4px solid #26BCB9;
+
+}
+.left-nav a:hover > i {
```
Comment (perryhoffman) [inline Mayra/css/styles.css:82]: Nice use of the first child selector!

**Review by perryhoffman — CHANGES_REQUESTED** [review]
Looking really good Mayra! I only have a couple of small suggestions/updates for you. Keep up the good work 👍

File: Mayra/css/styles.css [hunk 9]
```diff
@@ -13,15 +13,17 @@ header {
   float: left;
   height: 100%;
 }
-.btn {
+button {
```
Comment (perryhoffman) [inline Mayra/css/styles.css]: Careful, this gives all `button`'s on your site the same style. Was that your intent?

**Review by perryhoffman — APPROVED** [review]
👍

**Comment by perryhoffman** [comment]
Hiiiii @maiix !! 👋  😆

**Comment by maiix** [comment]
LOL hi @perryhoffman 👋 👋  I'm playing with my github account 😃

**Pull request closed without merging**